src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public interface Figure extends Flatness, Info {
```

Comment:
what's the purpose of aggregating these two interfaces into figure? implement them in abstract figure directly, it feels like this interface is redundant